src/index.ts

```diff
@@ -21,11 +22,24 @@ const DEFAULT_OPTIONS: IOptions = {
   isNativeRunnable: false
 }
 
-export function parseError(errors: SourceError[]): string {
+// deals with parsing error objects and converting them to strings (for repl at least)
+
+let verboseErrors = false
+
+export function parseError(errors: SourceError[], verbose: boolean = verboseErrors): string {
   const errorMessagesArr = errors.map(error => {
     const line = error.location ? error.location.start.line : '<unknown>'
+    const column = error.location ? error.location.start.column : '<unknown>'
     const explanation = error.explain()
-    return `Line ${line}: ${explanation}`
+    const elaboration = error.elaborate()
+
+    if (verbose) {
+      // TODO currently elaboration is just tagged on to a new line after the error message itself. find a better
+      // way to display it.
+      return `Line ${line}, Column ${column}: ${explanation}. ${elaboration}\n`
```

Comment:
I think your \n needs to be placed behind the elaboration not after it.

Reply:
I'd be against removing the '\n', since the intention was to separate the stack trace from the error message itself, to enhance readability. I'll add another '\n' between the explanation and elaboration.